processor/src/test/java/org/mapstruct/ap/test/defaultvalue/CountryMapper.java

```diff
@@ -40,19 +41,19 @@
             @Mapping( target = "code", defaultValue = "DE" ),
             @Mapping( target = "id", defaultValue = "42" ),
             @Mapping( target = "zipcode", defaultValue = "1337" ),
-            @Mapping( target = "region", ignore = true )
-
+            @Mapping( target = "region", ignore = true ),
+            @Mapping( target = "continent", defaultValue = "EUROPE" )
```

Comment:
earth?, solar system? galaxy? local cluster?.. We still have room for more patches :smile: